File: Python Code/Daily_Flash_Code_0/Week 5/10 Feb/24-DailyFlash_Solutions/24-DailyFlash_Solutions/10_Feb_Solutions_Three/Python/prog1.py
n = int(input("Enter num : "))
s = 0
i = 1
for j in range(1, n + 1):
    s = 0
    i = 1
    while(i * i <= j):
        if(j % i == 0):
            s += i
        if(int(j / i) != i and j % int(j / i) == 0):
                if(i == 1):
                    i += 1
                    continue
                s += int(j / i)
        i += 1
    if(s > j):
        print(j, end = " ")

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Three/C/prog1.c
#include<stdio.h>

int main(void) {
	int inc, num = 1;
	printf("Enter inc : ");
	scanf("%d", &inc);
	for(int i = 0; i < 10; i++) {
		printf("%d  ", num);
		num += inc;
	}
	return 0;
}
